services/compute/compute.go

```diff
@@ -1225,6 +1242,46 @@ type AvailabilitySet struct {
 	VirtualMachines []*VirtualMachineReference
 }
 
+type PlacementGroup struct {
+	// ID
+	ID *string `json:"ID,omitempty"`
+	// Name
+	Name *string `json:"name,omitempty"`
+	// Type
+	Type PlacementGroupType `json:"type,omitempty"`
```

Comment:
just curious why this PG property is not in 'PlacementGroupProperties' struct.

Reply:
Moved it inside placement group properties